libs/wingc/src/type_check/jsii_importer.rs

```diff
@@ -664,23 +665,39 @@ impl<'a> JsiiImporter<'a> {
 
 		// Add constructor to the class environment
 		let jsii_initializer = jsii_class.initializer.as_ref();
-
 		if let Some(initializer) = jsii_initializer {
 			let mut fn_params = vec![];
 			if let Some(params) = &initializer.parameters {
-				for (i, param) in params.iter().enumerate() {
-					// If this is a resource then skip scope and id arguments
-					// TODO hack - skip this check if the resource's name is "App"
-					// https://github.com/winglang/wing/issues/1485
```

Comment:
Does this PR add support for #1485? If not, can you add a note on #1485 saying what's still missing?

Reply:
Tested and it does seem to resolve #1485, good catch.